Providers/Profile.swift

```diff
@@ -444,7 +444,8 @@ open class BrowserProfile: Profile {
     }
 
     public func sendItems(_ items: [ShareItem], toClients clients: [RemoteClient]) -> Deferred<Maybe<SyncStatus>> {
-        let id = DeviceInfo.clientIdentifier(self.prefs)
+        let scratchpadPrefs = self.prefs.branch("sync.scratchpad")
+        let id = scratchpadPrefs.stringForKey("clientGUID")!
```

Comment:
We should avoid force unwraps. If the `clientGUID` is nil this would crash. We should make sure we still fall back to what we were using before

Reply:
something like. `let id  = scratchpadPrefs.stringForKey("clientGUID") ?? DeviceInfo.clientIdentifier(self.prefs)`

Reply:
`DeviceInfo.clientIdentifier(self.prefs)` did essentially nothing at all, it may as well have been a `""`. I think it's a little misleading to have it as the fallback.